File: Programmers/JAVA/lv1/lv1_문자열 내림차순으로 배치하기.java
https://programmers.co.kr/learn/courses/30/lessons/12917

import java.util.*;

class Solution {
    public String solution(String s) {
        String answer = "";
        char[] arr = s.toCharArray();
        Arrays.sort(arr);

        StringBuilder sb = new StringBuilder(String.valueOf(arr));
        answer = sb.reverse().toString();
        
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_문자열 내림차순으로 배치하기.cs
https://programmers.co.kr/learn/courses/30/lessons/12917?language=csharp

using System;

public class Solution
{
    public string solution(string s)
    {
        string answer = "";

        char[] temp = s.ToCharArray();
        Array.Sort(temp);
        Array.Reverse(temp);
        answer = new string(temp);

        return answer;
    }
}